src/client/dfuse/ops/open.c

```diff
@@ -67,6 +67,7 @@ dfuse_cb_open(fuse_req_t req, fuse_ino_t ino, struct fuse_file_info *fi)
 		 * which pre-existed in the container.
 		 */
 
+		/* TODO: This probably wants reflowing to not reference ie_open_count */
```

Comment:
I have two other PRs that depend on this one and cover the same area, I can look at this issue in the round once the three of them have all landed.